docs/config.md

```diff
@@ -80,6 +80,7 @@ For a good example, see https://riot.im/develop/config.json.
 1. `defaultCountryCode`: The ISO 3166 alpha2 country code to use when showing
    country selectors, like the phone number input on the registration page.
    Defaults to `GB` if the given code is unknown or not provided.
+1. `settingDefaults`:  Defaults for settings that support the `config` level, as an object mapping setting name to value (note that the "theme" setting is special cased to the `default_theme` in the config file).
```

Comment:
line wrap

Reply:
also could use more detail about what is able to be included here, as I don't expect that people will know to dive into `Settings.js` and rip out the object keys.